plugins/obs-qsv11/QSV_Encoder.cpp

```diff
@@ -68,6 +68,10 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 	blog(level, "[qsv encoder: '%s'] " format, \
 			"msdk_impl", ##__VA_ARGS__)
 
+#define WARN_ERR_IMPL(err, str, err_name) case err: \
+	do_log(LOG_WARNING, str " (" err_name ")"); break;
+#define WARN_ERR(err, str) WARN_ERR_IMPL(err, str, #err)
```

Comment:
Not super important in this case I guess, but this file is kind of on the verge of being big enough that you want to define those macros very close to their actual call sites (and then probably undefined them immediately after the switch, more or less)